src/content/blog/2024-12-04-differences-between-linting-and-type-checking.md

```diff
@@ -84,7 +84,7 @@ Linters, on the other hand, run with a configurable set of lint rules.
 Each lint rule can be granularly configured.
 If you don't like a particular lint rule, you can always turn it off for a line, set of files, or your entire project.
 
-Linters can also can be augmented by **plugins** that add new lint rules.
+Linters can also be augmented by **plugins** that add new lint rules.
 Plugin-specific lint rules extend the breadth of code checks that linter configurations can pick and choose from.
```

Comment:
Again, this is how ESLint works, not how all linters work. Some linters don't allow configuration at all. Most don't allow plugins.